packages/tizen_app_manager/tizen/src/tizen_app_manager_plugin.cc

```diff
@@ -157,30 +159,28 @@ void TizenAppManagerPlugin::GetApplicationInfo(
     LOG_ERROR("Failed to get app_info handler : %s", err_msg);
     result->Error(std::to_string(ret), "Failed to get app_info handler.",
                   flutter::EncodableValue(std::string(err_msg)));
-    goto cleanup;
+    if (app_info) {
+      app_info_destroy(app_info);
+    }
+    return;
   }
 
   ret = application_utils::GetAppData(app_info, value);
+  app_info_destroy(app_info);
   if (ret != APP_MANAGER_ERROR_NONE) {
     err_msg = get_error_message(ret);
+    LOG_ERROR("Failed to get app info : %s", err_msg);
     result->Error(std::to_string(ret), "Failed to get app info.",
                   flutter::EncodableValue(std::string(err_msg)));
-    goto cleanup;
+    return;
   }
-
   result->Success(flutter::EncodableValue(value));
-
-cleanup:
-  if (app_info) {
-    app_info_destroy(app_info);
-  }
 }
 
 void TizenAppManagerPlugin::ApplicationIsRunning(
     const flutter::EncodableValue &arguments, MethodResultPtr result) {
   std::string id = "";
   const char *app_id;
-  char *err_msg;
   bool running;
```

Comment:
```suggestion
  bool running = false;
```

Reply:
Please initialize local variables of `GetApplicationInfo` too.

Reply:
Initialized the parts you mentioned. Thank you!